tests/contrib/tornado/test_executor_decorator.py

```diff
@@ -44,7 +44,7 @@ def test_on_executor_handler(self):
 
         # this trace is executed in a different thread
         executor_span = traces[0][1]
-        assert "tornado-web" == executor_span.service
+        assert executor_span.service is None
```

Comment:
@erikayasuda I updated to do `assert ddtrace.config.service == executor_span.service` instead.

yes... it is still just asserting that it is `None`, but at least if the default service names change, or a global service name gets set, then this will still pass!